src/javascript/track.js

```diff
@@ -209,9 +209,9 @@ export class HitEffect {
     this.ctx = ctx;
   }
 
-  create(x, y, width, height, judge) {
-    x = x + width / 2 - 5;
-    width = 10;
+  create(mX, y, mWidth, height, judge) {
+    let x = mX + mWidth / 2 - 5;
+    let width = 10;
```

Comment:
A question: are you *certain* that nothing relies on some `x` from the outer-scope here being mutated by `create(...)`?

Reply:
:confused

Reply:
My bad, I didn't see that it was recycling the parameter...